platform/rct/Libraries/CMSIS/Device/ST/STM32F30x/Source/Templates/system_stm32f30x.c

```diff
@@ -201,11 +201,6 @@ void SystemInit(void)
        AHB/APBx prescalers and Flash settings ----------------------------------*/
     SetSysClock();
 
-#ifdef VECT_TAB_SRAM
-    SCB->VTOR = SRAM_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal SRAM. */
-#else
-    SCB->VTOR = FLASH_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal FLASH. */
```

Comment:
So where is our VTOR based out of now?  Base of FLASH?  Since that is where the bootloader lives?